web/src/components/views/CreateProjectFromFile.jsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react'
+
+import CreateProject from './CreateProject.jsx'
+
+const JSON5 = require('json5')
+
+export default class CreateProjectByURL extends React.Component {
+
+      constructor(props) {
+        super(props)
+        var data = this.props.json;
+        this.state = {
+          status : "2",
+          json : data
+        }
+      }
+
+      componentWillMount(){
+        if(this.props.match.path ==="/createFromFile/:data"){
+          var data = JSON5.parse(decodeURIComponent(this.props.match.params.data));
+          if(data!=null){
+            var project = {
+              title :"",
+              description: "",
+              date_creation: "2012-12-12",
+              status : "",
+              tags: [],
+              url: []
+            }
+            if(data.title!=null){
+              project["title"] = data.title
+            }
+            if(data.authors!=null){
+              var projectAuthors = []
+              for(var i = 0;i<data.authors.length;i++){
+                  projectAuthors.push(data.authors[i].name + " ("+data.authors[i].email + ")")
+              }
+              this.setState({ authors : projectAuthors })
+            }
+            if(data.date_creation!=null){
+              var projectDate = new Date(data.date_creation)
+              this.setState({date : projectDate})
+              project["date_creation"] = data.date_creation
```

Comment:
date_creation is okay.... but ...
date for the datepicker has to be defined as well